modules/ROOT/pages/keycloak/coding.adoc

```diff
@@ -9,74 +9,123 @@ This guide will help you to set up the {keycloak-service} service SDK in your Ap
 * xref:keycloak/provisioning.adoc[Provisioned {keycloak-service} service].
 * xref:keycloak/binding.adoc[Binding] created.
 * xref:keycloak/configuring-dev-env-keycloak.adoc[Configured dev environment].
-* xref:keycloak/keycloak-setup.adoc[Configured Keycloak].
+* Read the xref:configuring-dev-env.adoc[general app setup guide].
```

Comment:
Can we have this prerequisites in the same fashion that "Configuring a Dev environment"?